libs/geoservices/covid/src/lib/pages/maps/maps.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
+import { HttpClientModule } from '@angular/common/http';
+
+import { MapsMapboxModule } from '@tamu-gisc/maps/mapbox';
+import { UILayoutModule } from '@tamu-gisc/ui-kits/ngx/layout';
+import { UIFormsModule } from '@tamu-gisc/ui-kits/ngx/forms';
+
+import { MapsComponent } from './maps.component';
+import { TimeMapComponent } from './components/time-map/time-map.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: MapsComponent,
+    children: [
+      {
+        path: '',
+        pathMatch: 'full',
+        redirectTo: 'time-map'
+      },
+      {
+        path: 'time-map',
+        component: TimeMapComponent
+      }
+    ]
+  }
+]
+
+@NgModule({
+  declarations: [MapsComponent, TimeMapComponent],
+  imports: [
+    CommonModule, RouterModule.forChild(routes), MapsMapboxModule, HttpClientModule,UILayoutModule, UIFormsModule
```

Comment:
Please remember to trigger your formatter on every file changed. Space should be found between `HttpClientModule` and `UILayoutModule`. Committing and pushing these files consistently formatted will prevent white space changes on files across contributors.

Reply:
enabled the formatter, the next push will reflect Prettier formatting.